rdr_service/rdr_client/pairing_assigner.py

```diff
@@ -2,8 +2,8 @@
 
 Usage:
 ./rdr_client/run_client.sh --project all-of-us-rdr-prod --account $USER@pmi-ops.org \
-  pairing_assigner.py file.csv --pairing [site|organization|awardee] \
-  [--dry_run] [--override_site]
+  pairing_assigner.py file.csv --pairing [site|organization|awardee|biospecimen|physical_measurements] \
```

Comment:
pm & b are supposed to be optional flags, not finalization. we shouldn't be changing how we pair from site, org or awardee. Only updating the additional fields if --biospecimen flag, --physical_measurements flag, or both. 

Finalization should be updated automatically if these flags are added, no need for an optional flag for finalization.